great-idea/css/index.css

```diff
@@ -63,4 +63,254 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+
+* {
+	box-sizing: border-box;
+}
+
+p {
+	font-family: 'Roboto', sans-serif;
+}
+
+/* nav bar */
+
+body {
+	margin: 0 auto;
+	width: 876px;
+	display: block;
+	position: relative;
+}
+
+body nav {
+	position: fixed;
+	padding: 2rem 0;
+	top: 0;
+	background-color: white;
+	z-index: 500;
+	border-bottom: 1px solid grey;
+	width: 876px;
+}
+
+.navcontainer {
+	position: relative;
+	display: block;
+}
+
+.navmenu {
+	display: inline-block;
+	text-align: center;
+
+}
+
+.navmenu a {
+	padding: 0 3rem 1.7rem 0;
+	text-decoration: none;
+	color: grey;
+}
+
+.navmenu a:hover {
+	background-color: black;
+	color: white;
+}
+
+.navcontainer img {
+	display: inline-block;
+	position: absolute;
+	right: 0;
+	bottom: -6px;
+}
+
+/* .navmenu a:hover{
+
+} */
+
+/* header */
+
+.headermain {
+	position: relative;
+	padding: 3.5rem 0 23rem 0;
+	margin: 5rem 0 5rem 0;
+}
+
+.headertext {
+	display: inline-block;
+	text-align: center;
+	position: absolute;
+	left: 5rem;
+	top: 5.8rem;
+}
+
+.headertext h1 {
+	font-size: 5rem;
+}
+
+.headertext p {
+	display: inline-block;
+	border: 1px solid black;
+	padding: .6rem 3rem;
+	margin-top: 5px;
+}
```

Comment:
It's not best practice to use rem and px in the same stylesheet. I suggest digging into rem, em and px to fully understand their use in terms of accessibility. I like that you're utilizing rem already its a more advanced concept.

Reply:
not sure if I should refractor this one, I did mix and match because rems is a bit more complex and I found px to be a quick fix but I'll keep that in mind next time.